_data-prepper/pipelines/configuration/sources/s3.md

```diff
@@ -20,14 +20,14 @@ codec | Yes | Codec | The codec to apply. Must be `newline`, `json`, or `csv`.
 sqs | Yes | sqs | The [Amazon Simple Queue Service](https://aws.amazon.com/sqs/) (Amazon SQS) configuration. See [sqs](#sqs) for details.
 aws | Yes | aws | The AWS configuration. See [aws](#aws) for details.
 on_error | No | String |  Determines how to handle errors in Amazon SQS. Can be either `retain_messages` or `delete_messages`. If `retain_messages`, then Data Prepper will leave the message in the SQS queue and try again. This is recommended for dead-letter queues. If `delete_messages`, then Data Prepper will delete failed messages. Default is `retain_messages`.
-buffer_timeout | No | Duration | The timeout for writing events to the Data Prepper buffer. Any events that the S3 Source cannot write to the buffer in this time will be discarded. Default is 10 seconds.
+buffer_timeout | No | Duration | The timeout for writing events to the Data Prepper buffer. Any events that the S3Source cannot write to the buffer in this time will be discarded. Default is 10 seconds.
 records_to_accumulate | No | Integer | The number of messages that accumulate before writing to the buffer. Default is 100.
 metadata_root_key | No | String | Base key for adding S3 metadata to each Event. The metadata includes the key and bucket for each S3 object. Defaults to `s3/`.
-disable_bucket_ownership_validation | No | Boolean | If `true`, then the S3 Source will not attempt to validate that the bucket is owned by the expected account. The only expected account is the same account that owns the SQS queue. Defaults to `false`.
+disable_bucket_ownership_validation | No | Boolean | If `true`, then the S3Source will not attempt to validate that the bucket is owned by the expected account. The only expected account is the same account that owns the SQS queue. Defaults to `false`.
 
 ## sqs
 
-The following are configure usage of Amazon SQS in the S3 Source plugin.
+The following are configure usage of Amazon SQS in the S3Source plugin.
```

Comment:
Could we clarify this sentence? I'm not sure what "configure usage" means...

Reply:
If it helps our understanding of S3Source, see the QuickSight documentation about S3Source https://docs.aws.amazon.com/quicksight/latest/APIReference/API_S3Source.html